src/modules/acceptance-tests/routes.js

```diff
@@ -0,0 +1,27 @@
+const controller = require('./controller');
+const { version, isProduction } = require('../../../config');
+
+const routes = {
+
+  postSetup: {
+    method: 'POST',
+    path: `/water/${version}/acceptance-tests/set-up`,
+    handler: controller.postSetup,
+    config: {
+      description: 'Creates the required data to allow acceptance tests to run'
+    }
+  },
+
+  postTearDown: {
+    method: 'POST',
+    path: `/water/${version}/acceptance-tests/tear-down`,
+    handler: controller.postTearDown,
+    config: {
+      description: 'Deletes an data created for use with acceptance tests'
+    }
+  }
+};
+
+if (!isProduction) {
```

Comment:
I wonder if:
```
(isTestMode)
// or
(isTestMode || isPreProduction)
```
Would be safest here depending where the test suite needs to run - as we had issues in the past with blacklisting environments rather than whitelisting for test flags.